File: Java/TP3-1/src/Livro.java

public class Livro {

	private String nome, autor;
	private int anoPublicacao;

	public Livro(String nome, String autor, int anoPublicacao) {
		this.nome = nome;
		this.autor = autor;
		this.anoPublicacao = anoPublicacao;
	}

	public String getNome() {
		return nome;
	}

	public String getAutor() {
		return autor;
	}

	public int getAnoPublicacao() {
		return anoPublicacao;
	}

	public void mostrarInfoLivro() {
		System.out.println("Livro: " + getNome());
		System.out.println("Autor: " + getAutor());
		System.out.println("Ano de Publicação: " + getAnoPublicacao());
	}
	
	@Override
	public String toString() {
		return "\nLivro: " + getNome() + "\nAutor: " + getAutor() + "\nAno de publicacao: " + getAnoPublicacao();
	}

	
}

File: Kotlin/TPK3-1/bin/Livro.kt
class Livro(nome: String, autor: String, anoPublicacao: Int) {
	var nome: String
	var autor: String
	var anoPublicacao: Int = 0

	init {
		this.nome = nome
		this.autor = autor
		this.anoPublicacao = anoPublicacao
	}
	fun getNome(): String {
		return nome
	}
	
	fun getAutor(): String {
		return autor
	}
	
	fun getAnoPublicacao(): Int {
		return anoPublicacao
	}

	fun mostrarInfoLivro() {
		println("Livro: " + nome)
		println("Autor: " + autor)
		println("Ano de Publicação: " + anoPublicacao)
	}

	public override fun toString(): String {
		return "\nLivro: " + nome + "\nAutor: " + autor + "\nAno de publicacao: " + anoPublicacao
	}
}